evap/staff/urls.py

```diff
@@ -76,5 +76,7 @@
     path("faq/", views.faq_index, name="faq_index"),
     path("faq/<int:section_id>", views.faq_section, name="faq_section"),
 
-    path("download_sample_xls/<str:filename>", views.download_sample_xls, name="download_sample_xls")
+    path("download_sample_xls/<str:filename>", views.download_sample_xls, name="download_sample_xls"),
+
+    path("development/components", views.development_components, name="development_components"),
```

Comment:
sooo... the idea is that e.g. when someone is changing stuff in the scss files, they'd primarily check this page whether everything is fine?

Reply:
That and it is the place to go when defining new elements or making decisions on colors etc.

Reply:
yeah, nice. I like it 👍